### ISSUE TITLE
Warnings about telegram, although telegram is not enabled

### ISSUE THREAD

**Issue by web2brain** [Der-Henning/tgtg#444]
Since the last update, I get log entries about telegram, although it is not actvated. Setting the telegram integration to `false` obviously didn't change this as `false` already is the default:

```2024-01-12 16:39:54 WARNING  You enabled the Telegram notifications without providing a chat id!
2024-01-12 16:39:54 WARNING  Send **** to the bot in your desired chat.
2024-01-12 16:39:54 WARNING  Waiting for code ...
2024-01-12 16:39:54 WARNING  Error starting Telegram: [] - You must pass the token you received from https://t.me/Botfather!```

**Comment by Der-Henning** [comment]
Hi @web2brain.
I just tried but couldn't reproduce your issue. Can you please provide some more information? 
Which version on which OS are you using? Are you running the executable, docker, or something else? 
Could you also please provide your configuration? With masked tokens and personal information of course.

**Comment by web2brain** [comment]
Hi @Der-Henning,
sorry, I should have been more specific.

I use the alpine docker container on a raspberry pi with the following docker-compose.yml:

```version: '3.3'

 services:
   scanner:
     image: derhenning/tgtg:latest-alpine         ## pre build image from docker hub
     container_name: tgtg
     #image: tgtg-scanner:latest-alpine           ## locally build image
     environment:
     # Configuration via environment variables.
     # For more options and details visit https://github.com/Der-Henning/tgtg/wiki/Configuration

     - TGTG_USERNAME=email@domain.com
     - SLEEP_TIME=60
     - TZ=Europe/Berlin
     - LOCALE=de_DE

     - NTFY=true
     - NTFY_SERVER=https://url.com
     - NTFY_TOPIC=xyz
     - NTFY_USERNAME=abc
     - NTFY_PASSWORD=def

     volumes:
     - tokens:/tokens             ## volume to save TGTG credentials to reuse on next start up and avoid login mail

 volumes:
   tokens:
```

**Comment by Der-Henning** [comment]
The Bug should be fixed in Release 1.19.0.

**Comment by web2brain** [comment]
Thanks @Der-Henning
